content/styles/03-components/header.css

```diff
@@ -18,6 +18,15 @@ header {
       font-weight: 300;
       font-size: 3rem;
 
+      &:hover,
+      &:focus,
+      &:active {
+        a {
+          cursor: pointer;
```

Comment:
why do we need a pointer? If it's a link it should be a pointer automatically. 🤔

Reply:
Thanks for the catch, addressed with 2b6fe17e35f48008ea376403e8d8dc8630e5d5fc